src/style.css

```diff
@@ -0,0 +1,20 @@
+* {
+  padding: 0;
+  margin: 0;
```

Comment:
You dont need to reset this in all elements

Reply:
changed reset only div